examples/networkpolicy/kubernetes-azure-hybrid.json

```diff
@@ -0,0 +1,52 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "networkPolicy": "azure"
+      }
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "",
+      "vmSize": "Standard_D2_v2"
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "linuxpool1",
+        "count": 2,
+        "vmSize": "Standard_D2_v2",
+        "availabilityProfile": "AvailabilitySet",
+        "ipAddressCount": 20,
+        "osType": "Linux"
+      },
+      {
+        "name": "windowspool2",
+        "count": 2,
+        "vmSize": "Standard_DS3_v2",
+        "availabilityProfile": "AvailabilitySet",
+        "ipAddressCount": 30,
+        "osType": "Windows"
```

Comment:
Isn't this going to hit the validation error?

The intention of the request is to ensure this change works on > 1 agent pool.  So if you don't want to put a multi linux agent pool in the examples folder, put in the test folder.